mysite/_config/routes.yml

```diff
@@ -0,0 +1,7 @@
+---
+Name: mysiteroutes
+After: framework/routes#coreroutes
+---
+Director: 
+    rules:
+        '/RestController/$Action//$ID': 'RestController'
```

Comment:
try
```
Director:
  rules:
    'RestController/$Action//$ID': 'RestController'
```

Reply:
Also you probably want to expand your rule with
'RestController/$Action//$ID/$ClassName so that you can pass in the classname of what you want to delete.

Reply:
Yea I at first added it by then I thought that was the problem so I deleted it to test it out first.